osu.Desktop/Program.cs

```diff
@@ -169,6 +169,14 @@ private static bool trySendIPCMessage(IIpcHost host, string cwd, string[] args)
 
         private static void setupVelopack()
         {
+            string? packageManaged = Environment.GetEnvironmentVariable("OSU_EXTERNAL_UPDATE_PROVIDER");
```

Comment:
Can this be moved to a common (static) location so that it's not referenced twice by name in two different places?